### ISSUE TITLE
Error using multiple server/repos

### ISSUE THREAD

**Issue by philkohl** [coderazzi/aws_codeartifact_maven#10]
Hi, 

at first thanks for you plugin! It makes it easier to work with the AWS Codeartifact :)
At the moment we face an issue, if we work with several servers/profiles

```
<server>
   <id>server1</id>
   <username>user1</username>
   <password></password>
</server>
<server>
   <id>server1-snapshots</id>
   <username>user1</username> <!-- Same user as the first one -->
   <password></password>
</server>
<profiles>
        <profile>
            <id>server1</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <repositories>
                <repository>
                    <id>server1</id>
                    <url>...</url>
                    <snapshots>
                        <enabled>false</enabled>
                    </snapshots>
                    <releases>
                        <enabled>true</enabled>
                    </releases>
                </repository>
                <repository>
                    <id>server1-snapshots</id>
                    <url>...</url>
                    <snapshots>
                        <enabled>true</enabled>
                    </snapshots>
                    <releases>
                        <enabled>false</enabled>
                    </releases>
                </repository>
            </repositories>
        </profile>
    </profiles>
```

I can generate the credentials for server1, but if I try this for server1-snapshots, I get the following error:
![image](https://github.com/coderazzi/aws_codeartifact_maven/assets/65112414/cf4698a0-7e4b-433e-b696-65b74514d3ca)
I also tried to separate the repositories to two different profiles, but that does not do the trick.
Do I have to configure it differently?

Based on https://github.com/coderazzi/aws_codeartifact_maven/issues/3 I think your plugin should support it?

**Comment by coderazzi** [comment]
Hi, could you send me your settings.xml and .aws folder, removing any
sensitive information?

**Comment by philkohl** [comment]
[aws_codeartifact_maven.zip](https://github.com/coderazzi/aws_codeartifact_maven/files/12647697/aws_codeartifact_maven.zip)

Sure, I attached the two .m2 settings that I tried. The aws config is very basic. Thanks for taking a look!

**Comment by coderazzi** [comment]
Hi, little, silly bug, ashamed to say. I will release tomorrow asap
the solution ....

**Comment by coderazzi** [comment]
Hi, I have released version 3.2.3, covering the bug you found.
You could download it from the beta channel and test it:
https://plugins.jetbrains.com/plugin/16777-aws-codeartifact--maven/versions/beta
Best,
 Lu

**Comment by philkohl** [comment]
Hi, thank you very much for the fix! Now it's working :)
